src/table/TableBodyWrapper.jsx

```diff
@@ -57,18 +90,43 @@ const TableBodyWrapper = ({
     colIdx: -1,
     isEnabled: selectionsEnabled,
   });
+  const [isScrollbar, setIsScrollbar] = useState(false);
 
   useEffect(() => {
     selDispatch({ type: 'set-enabled', payload: { isEnabled: selectionsEnabled } });
     setColumnRenderers(getColumnRenderers);
   }, [selectionsEnabled, columns.length]);
 
+  // Setup isScrolling variable
+  let isScrolling;
+  // eslint-disable-next-line consistent-return
   useEffect(() => {
     addSelectionListeners(selectionsAPI, selDispatch, setShouldRefocus);
+    // to display scrolling bar when the user scrolls in windows
+    // if (navigator.appVersion.indexOf('Win') !== -1) {
+    const handleScroll = () => {
+      isScrollbar === false && setIsScrollbar(true);
+      // Clear our timeout throughout the scroll
+      window.clearTimeout(isScrolling);
+      // Set a timeout to run after scrolling ends
+      isScrolling = setTimeout(() => setIsScrollbar(false), 100);
```

Comment:
Is the timeout needed?

Reply:
I just feel it is better to have a timeout to disappear the scrollbar

Reply:
hmm yes I agree